tracers/logger/logger.go

```diff
@@ -446,7 +446,7 @@ func formatLogs(logs []StructLog) []StructLogRes {
 			}
 			formatted[index].Stack = &stack
 		}
-		if trace.ReturnData != nil && len(trace.ReturnData) > 0 {
```

Comment:
dont we need this check ?

Reply:
`ReturnData` is not a pointer, so its never null. It can only be it's zero value (empty byte array)